en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,80 @@
+
+- **Dependency alignment:** At launch, Silverstripe CMS major releases aim to:
+
+  - Support all PHP versions currently in full support.
+  - Have all PHP and JavaScript dependencies on their latest stable versions.
+
+- **Upgrade path:** Silverstripe CMS commits to providing a clear upgrade path to new major releases, including comprehensive documentation on all breaking changes and migration guides.
+
+### Minor releases `(x.y.0)`
+
+Minor releases provide ongoing value and improvements within a stable major release line.
+
+- **Purpose:** To ship new features, enhancements, performance improvements, and bug fixes that cannot be shipped in a patch releae due to API additions.
+
+- **Backward compatibility:** Minor releases are **backward compatible** within their major release line. They will not introduce breaking changes.
+
+- **Frequency:** Minor releases are shipped every **six months** within an active major release line, typically targeted for **April and October**. There will be **five** minor releases per major release.
+
+- **Pre-release period:** Minor releases are typically preceded by a **beta period of approximately six weeks** and a **release candidate (RC) period of approximately two weeks**. A feature freeze is implemented after the beta phase.
+
+### Patch releases `(x.y.z)`
+
+Patch releases address critical issues swiftly to maintain stability and security.
+
+- **Purpose:** To deliver bug fixes and security updates.
+
+- **Backward compatibility:** Patch releases are **backward compatible** within their major and minor release line. They will not introduce new features or breaking changes.
+
+- **Frequency:** Patch releases are shipped on an as-needed basis in response to identified bugs or security vulnerabilities. They do not follow a fixed schedule.
+
+## Support lifecycle
+
+Silverstripe CMS maintains a clear support lifecycle for versions that have a stable releases to provide long-term stability and planning for users. Each minor release version progresses through defined support phases:
+
+- **Full support:** Receives bug fixes and updates for all indentified security vulnerabilies.
+
+- **Partial support:** Receives no bug fixes. Receives security updates for high and critical security vulnerabilities only (defined as those with a **CVSS score of 7.0 or above**). Low and medium level security vulnerabilities are not fixed.
+
+- **End-of-life (EOL):** These minor versions receive no further support, bug fixes, or security updates. Users are strongly encouraged to upgrade to a supported version.
+
+Most minor releases start with **six months** of full support which is followed by **six months** of partial support. The last minor release of major version, typically the X.4 release, has an extended support length which is **one year**.of full support which is followed by **one year** of partial support.
```

Comment:
```suggestion
```
I think it's cleaner to outline this information inside sections for each support phase.

The fact that the last minor release of a given major is the one with extended support should either be at the start of the lifecycle section instead of the end. I'll add a suggestion there with some wording.